Exercises/this/this.js

```diff
@@ -0,0 +1,46 @@
+// Task 1,2
+let student = {
+    name: "Kate",
+    birthYear: 1987,
+    age: function () {
+        const currentYear = new Date().getFullYear()
+        return currentYear - this.birthYear;
+    },
+}
+console.log(student.age());
+const defineAge = student.age;
+console.log(defineAge());
+console.log(defineAge.call(student));
+
+// Task 3
+let graduate = {
+    name: "Alex",
+    birthYear: 1979,
+}
+console.log(defineAge.call(graduate));
+
+// Task 4
+let a = {
+    name: "John",
+}
+let b = {
+    name: "Peter",
+    greeting: function () {
+        console.log(`Hi, ${a.name}`);
+    }
```

Comment:
Here using of arrow function was meant, just like in task 5. You need to get access to external `this`, not to external variables. You always have an access to external variables so it would be too easy)